plugins/frontend/backstage-plugin-shortcut/src/components/Widgets/StoriesCard.tsx

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  Box,
+  Grid,
+  Link,
+  makeStyles,
+  Tooltip,
+  Typography,
+} from '@material-ui/core';
+import { InfoCard, Progress } from '@backstage/core-components';
+import { Story } from '../../api/types';
+import { useAsync } from 'react-use';
+import { identityApiRef, useApi } from '@backstage/core-plugin-api';
+import { Alert } from '@material-ui/lab';
+import BugReport from '@material-ui/icons/BugReport';
+import Star from '@material-ui/icons/Star';
+import Build from '@material-ui/icons/Build';
+import { shortcutApiRef } from '../../api';
+import { BackstageTheme } from '@backstage/theme';
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  listStyle: {
+    borderRadius: '1rem',
+    boxShadow:
+      '5px 0 5px -5px rgb(0 0 0 / 70%), -5px 0 5px -5px rgb(0 0 0 / 70%)',
```

Comment:
there are boxShadows in the theme. Can you check if there is one that suits this shadow? If not it's ok.
for example:
```
theme.shadows[10]
```

Reply:
Yeah, tbh I didn't find a way to test them without trying each one of them so I just went straight in with this. I will check 😄

Reply:
Yea, it is cumbersome, I'd love for our plugins to use the most CSS from the `theme` so when they get implemented in a backstage they would still look consistent with their setup